.github/workflows/e2e_test.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: |
           git clone --branch "${{ env.TERRAFORM_REPO_VERSION }}" "${{ env.TERRAFORM_REPO }}" terraform-repo
           
-          cd terraform-repo/${{ .github.event.inputs.path_to_installation }}
+          cd terraform-repo/${{ github.event.inputs.path_to_installation }}
```

Comment:
Why you dont want use here actions/checkout ?

Reply:
updated to use checkout action